src/Symfony/Component/Routing/Tests/Fixtures/AttributeFixtures/InvokableFQCNAliasConflictController.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Symfony\Component\Routing\Tests\Fixtures\AttributeFixtures;
+
+use Symfony\Component\Routing\Attribute\Route;
+
+#[Route(path: '/foobarccc', name: self::class)]
```

Comment:
I added only this test fixture case but the 3 FQCN aliases are susceptible to fail.